lib/Rex/Commands/Iptables.pm

```diff
@@ -548,4 +610,25 @@ sub _rule_exists {
   return 0;
 }
 
+sub _get_ip_version {
+  my ($params) = @_;
+  if ( defined $params->[0] && !ref $params->[0] ) {
+    if ( $params->[0] eq "-4" || $params->[0] eq "-6" ) {
+      return shift @$params;
+    }
+  }
+  return -4;
+}
+
+sub _get_executable {
+  my ($params) = @_;
+  return _get_ip_version($params) == -6 ? "ip6tables" : "iptables";
+}
+
+sub _iptables_version {
+  my $version = join "", map { sprintf "%03d", $_ } split /[.]/,
+    substr run("iptables -V"), 10;
```

Comment:
If you need to `run` something for internal use, please use `i_run` for that.

Do you think we can simplify this version retrieval? It looks correct, but it also seems like doing a lot of hoops just to retrieve, parse and normalize the version string (which can be a PITA, I know..).

Do we need to check for `iptables` or `ip6tables` version (i.e. calling `_get_executable` first to see what needs checking) depending on which mode is requested?